File: Javascript/InferenceHosted.js
const axios = require("axios");

// Model Endpoint
var model_endpoint = "dataset/v";
// Enter your API Key Here
var api_key = "";

axios({
  method: "POST",
  url: "https://detect.roboflow.com/" + model_endpoint,
  params: {
    api_key: api_key,
    image: "https://i.imgur.com/PEEvqPN.png",
  },
})
  .then(function (response) {
    console.log(response.data);
  })
  .catch(function (error) {
    console.log(error.message);
  });

File: Ruby/InferenceHosted.rb
require 'httparty'
require 'cgi'

model_endpoint = "dataset/v" # Set model endpoint
api_key = "" # Your API KEY Here
img_url = "https://i.imgur.com/PEEvqPN.png" # Construct the URL

img_url = CGI::escape(img_url)

params =  "?api_key=" + api_key + "&image=" + img_url

response = HTTParty.post(
    "https://detect.roboflow.com/" + model_endpoint + params,
    headers: {
    'Content-Type' => 'application/x-www-form-urlencoded',
    'charset' => 'utf-8'
  })

puts response